packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -936,19 +939,18 @@ public void WriteArguments(IEnumerable<ValueExpression> arguments, bool useSingl
             }
             else
             {
-                WriteRawLine("(");
+                AppendRaw("(");
```

Comment:
Can we add a test to validate the multiline arg case here?  I think you need more than 6 args to get into this else or explicitly set usesingleline false